lms/djangoapps/onboarding_survey/models.py

```diff
@@ -2,7 +2,38 @@
 Models to support the on-boarding surveys
 """
 from django.db import models
+from django import forms
 from django.contrib.auth.models import User
+from student.models import UserProfile
+
+
+class Organization(models.Model):
+    """
+    Represents an organization.
+    """
+
+    name = models.CharField(max_length=255, db_index=True)
+    created = models.DateTimeField(auto_now_add=True, null=True, db_index=True)
+    point_of_contact_exist = models.BooleanField(default=False)
+
+    def __str__(self):
+        return self.name
+
+
+class ExtendedProfile(models.Model):
+    """
+    Model for extra fields in registration form.
+    """
+    POC_CHOICES = ((1, 'Yes'), (0, 'No'))
+
+    first_name = models.CharField(max_length=256)
+    last_name = models.CharField(max_length=256)
+    organization = models.ForeignKey(Organization, related_name='extended_profile', blank=True, null=True)
```

Comment:
related_name='extended_profiles'
where is cascase with SET_NULL?

Reply:
Done